src/chief.sol

```diff
@@ -35,6 +35,12 @@ contract DSChiefApprovals is DSThing {
 
     uint256 public MAX_YAYS;
 
+    mapping(address=>uint256) public last;
+
+    bool public live;
+
+    uint256 constant LAUNCH_THRESHOLD = 100000 * 10 ** 18; // 100K MKR launch threshold
```

Comment:
Everyone else can give approvals, but let me get the final answer on this amount.

Reply:
Okay, myself, LFW and the other mandated actors think we should set this to `80000` MKR for activation.